lib/client/ca_export.go

```diff
@@ -423,9 +392,33 @@ func hostCAFormat(ca types.CertAuthority, keyBytes []byte, client authclient.Cli
 	})
 }
 
-func exportAuthForIntegration(ctx context.Context, client authclient.ClientI, req ExportAuthoritiesRequest) ([]*ExportedAuthority, error) {
-	switch req.AuthType {
-	case "github":
+// IsIntegrationAuthorityType returns true if provided type is an integration CA
+// type.
+func IsIntegrationAuthorityType(exportType string) bool {
+	return exportType == types.IntegrationSubKindGitHub
+}
+
+// ExportIntegrationAuthoritiesRequest has the required fields to create an
+// export authorities request for integrations.
+type ExportIntegrationAuthoritiesRequest struct {
+	// Type is the integration type.
+	Type string
+	// MatchFingerprint filters authorities using provided fingerprint if
+	// specified.
+	MatchFingerprint string
+	// Integration is the name of the integration.
```

Comment:
Add an example too, if possible?

Reply:
changed the comment to `// Integration is the name of the integration resource.`. This is the actual resource name so not sure what an example can show.

Reply:
Thanks, it's all much clearer after the comments.